src/neptune_fetch_api/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""A client library for accessing Neptune Leaderboard REST API"""
```

Comment:
We need to figure out better naming & package structure, as leaderboard API has non-fetch calls as well, so it's not read-only. Ideally all code generated from various swagger files should sit in `neptune_api`, with submodules like `neptune_api.leaderboard`, `neptune_api.backend` etc